packages/agoric-cli/lib/start.js

```diff
@@ -487,20 +496,41 @@ export default async function startMain(progname, rawArgs, powers, opts) {
               soloAddr,
               ...provisionPowers,
             ],
-            true,
-          );
-          // eslint-disable-next-line no-await-in-loop
-          exitStatus = await capret[0];
-          if (!exitStatus) {
-            const json = capret[1].replace(/^gas estimate: \d+$/m, '');
-            try {
-              const ret = JSON.parse(json);
-              if (ret.code !== 0) {
-                exitStatus = 2;
+            // Then send it some coins.
+            [
+              'tx',
+              'bank',
+              'send',
+              '--keyring-backend=test',
```

Comment:
where does the key itself come from?

Reply:
The key is named "provision", on line 511.